### ISSUE TITLE
No lists in footnotes permitted

### ISSUE THREAD

**Issue by mkraetke** [transpect/evolve-hub#1]
In [`hub:lists-permitted-here()`](https://github.com/transpect/evolve-hub/blob/master/lists-by-indent/xsl/handle-indent.xsl#L44) is defined that lists shouldn't be created in footnotes. I wonder why this condition was defined, since you can render lists in footnotes with TeX and InDesign. As far as I know, it's no issue to render lists in footnotes (e.g. endnotes) in most EPUB reading systems, too.

**Comment by gimsieke** [comment]
I don’t think that we want to rule out lists in footnotes out of “content hygiene” considerations. Not considering footnotes in the list detection is (or was) probably necessitated by the list nesting mechanism, to which footnotes can look like list items.

If you look at [from what it evolved](https://github.com/transpect/evolve-hub/blame/99cee6d19840a0c6db68b4ae9d44522cde978120/lists-by-indent/xsl/handle-indent.xsl#L44), footnotes might have been excluded from list detection because they can have a margin and a negative indent that might make them look like list items to the converter. 

The version that I linked to was probably the one [imported from our public svn](https://subversion.le-tex.de/common/evolve-hub/lists-by-indent/handle-indent.xsl), so we’d need to dig into its history. It was added probably from [our internal svn](http://svn/svn/ltxbase/xsl-modules/lists/handle-indent.xsl) in r233.

The footnote condition was originally added to http://svn/svn/ltxbase/xsl-modules/lists/handle-indent.xsl by Moriz on 2011-06-30 with the comment “handle-indent nicht in footnote psoprocess added”, whatever this was intended to mean. 

If you consider removing this condition, I propose you run the tests “evolve-hub-lists” ([testset](http://svn.le-tex.de/svn/ltxbase/Difftestdata/evolve-hub-lists/)) locally first and see what happens.
